certify/crlset_parser.hpp

```diff
@@ -0,0 +1,73 @@
+#ifndef BOOST_CERTIFY_CRLSET_PARSER_HPP
```

Comment:
Are these certify files coming from a git repo? Perhaps we can arrange to insert them as a submodule.

Reply:
I just included the header folder of the repository: [https://github.com/djarek/certify/tree/master/include/boost/certify](https://github.com/djarek/certify/tree/master/include/boost/certify)

That was the reason I didn't add it as a submodule, because for our purposes it seems that we only need the headers.